### PR TITLE
GET /api/profile

### LINKED ISSUE #25: Implement GET /api/profile/  endpoint
- Implement GET /api/profile/
- Add documentation and unit testing.

### PR DESCRIPTION
**Pull request by irem-kurt**
Hey @muslimomar, I added unit testing as well. Everything is ready to be merged.

fix #25

### CODE REVIEW

File: src/controllers/__tests__/profile.test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+const {
+  connectToMongoAtlas,
+  closeDatabase,
+  clearDatabase,
+} = require('../../db/connection');
+
+const user = {
+  firstName: 'Glenn',
+  lastName: 'Quagmire',
```
Comment (muslimomar) [inline src/controllers/__tests__/profile.test.js:15]: Irem Quagmire 😂

**Review by muslimomar — APPROVED** [review]
Super!

**Comment by muslimomar** [comment]
@irem-kurt , merge conflict

File: src/controllers/__tests__/profile.test.js (hunk 1 above)
Comment (irem-kurt) [reply src/controllers/__tests__/profile.test.js:15]: hahaha I wished I wrote Irem QuagMERI 😄

**Comment by irem-kurt** [comment]
Hey @muslimomar , I solved conflict.

**Comment by muslimomar** [comment]
Waiting for the conflict

**Comment by irem-kurt** [comment]
Hi @muslimomar , I fixed conflicts by updating swagger and profile test which is stated here.

**Review by muslimomar — APPROVED** [review]

**Pull request merged**